packages/blade/src/components/Checkbox/Checkbox.tsx

```diff
@@ -165,6 +167,7 @@ const _Checkbox: React.ForwardRefRenderFunction<BladeElementRef, CheckboxProps>
   const _hasError = _validationState === 'error';
   const _isDisabled = isDisabled ?? groupProps?.isDisabled;
   const _name = name ?? groupProps?.name;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
Can we remove this rule altogether? @anuraghazra thoughts?

Reply:
We are disabling this rule in 5 places, let's keep it for now, can disable this if it becomes annoying.

Reply:
5 times already indicates that this not required? 😄